lib/queries/edit.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+var escape = require('pg-escape');
+
+var fields = ['title', 'description'];
+
+module.exports = function (id, obj) {
+  var newValues = fields.reduce(function (arr, key) {
+    if (obj[key]) {
+      return arr.concat(key + ' = ' + escape.literal(obj[key]));
+    }
+
+    return arr;
+  }, []).join(', ');
+
+  return [
+    'UPDATE',
+    'challenges',
+    'SET',
+    newValues,
```

Comment:
if the "obj" don't have any of the required values define in fields (title, description) I think the query will fail as it will look like this:
```
update challenges set where id = theId
```
We need to add a validation on the object to check that the object is well defined, are title and description required field, are there string, do we want a minimum length for the strings...